rotkehlchen/tests/api/test_liquity.py

```diff
@@ -249,6 +249,12 @@ def test_staking_stats(rotkehlchen_api_server, ethereum_accounts):
     information and that the stats combining all the data are consistent with the
     information for each tracked address
     """
+    response = requests.post(
```

Comment:
why add the query at start?

Reply:
To make sure that no event is left undecoded. For example if in a db upgrade we delete the decoded events

Reply:
Then please add a comment there in your next PR.